rbmc-cfam-daemon/src/cfam_tool.cpp

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: Apache-2.0
+#include "local_cfam.hpp"
+
+#include <CLI/CLI.hpp>
+
+#include <format>
+#include <iostream>
+#include <string_view>
+
+void displayCFAMFields(std::string_view name, size_t link)
+{
+    SysFSImpl sysfs;
+    LocalCFAM cfam{link, sysfs};
+
+    std::cout << std::format("{} CFAM scratchpad fields\n", name);
+
+    for (const auto& [field, meta] : LocalCFAM::cfamFields)
+    {
+        auto data = cfam.readField(field);
+        if (!data.has_value())
+        {
+            std::cerr << std::format("Unable to read {} CFAM ({})\n", name,
+                                     data.error());
+            return;
+        }
+
+        std::cout << std::format("{:24} {:#x}\n", meta.desc, data.value());
+    }
+}
+
+void displayCFAMs()
+{
+    displayCFAMFields("Local BMC", 0);
+    std::cout << '\n';
+    displayCFAMFields("Sibling BMC", 1);
+}
+
+int main(int argc, char** argv)
+{
+    CLI::App app{"CFAM Tool"};
```

Comment:
> CFAM Tool

I believe making it more specific will help avoid confusion with the other CFAM (the CEC one). Could we update the name to reflect the scope more clearly?

Perhaps something like `CFAM-S Tool` or `RBMC CFAM Tool`?

As for the tool name, maybe `cfamstool`?

Reply:
sure.